covalent_ec2_plugin/assets/infra/main.tf

```diff
@@ -78,7 +78,7 @@ resource "null_resource" "deps_install" {
       "echo 'Installing Covalent...'",
 
       # TODO: Update to a variable version
-      "pip install \"covalent${var.covalent_version}\"",
+      "pip install \"covalent==${var.covalent_version}\"",
```

Comment:
@mpvgithub the reason why this is like this, is because in some cases, it might be required to install the covalent version with `qiskit or braket` as well, see this line https://github.com/AgnostiqHQ/covalent-ec2-plugin/blob/14114f0300a59ba575771e0e7e7308c74a7e897e/covalent_ec2_plugin/ec2.py#L76.